app/assets/stylesheets/content/_tooltips.sass

```diff
@@ -182,3 +182,48 @@ table
       padding: 4px
       font-size: 0.8em
       color: #505050
+
+// tooltips with html support
+span.op_tooltip
+  visibility: hidden
+  opacity: 0
+  transition: visibility 0s linear 0.2s, opacity 0.2s linear
+  position: absolute
+  border: 2px solid #a7cbe1
```

Comment:
In general, for all colours, try to pull them out into variables, ideally, it should match up with the normal tooltips. I know there is a visual guide but these are not always consistent.

Please have a look at the tooltip variables for colours:\

```

$tooltip--background-color: #e3f5ff
$tooltip--border: none
$tooltip--height: 22px
$tooltip--font-color: $body-font-color
$tooltip--arrow-size: 6px
```

and try to reuse first.